dbms/src/Common/MPMCQueue.h

```diff
@@ -436,6 +472,7 @@ class MPMCQueue
     Int64 write_pos = 0;
     Status status = Status::NORMAL;
     String cancel_reason;
+    size_t current_auxiliary_memory_usage = 0;
```

Comment:
Use Int64 here to avoid incorrect get_auxiliary_memory_usage case?

Reply:
Done